zebra-rpc/src/methods/get_block_template_rpcs/types/get_block_template.rs

```diff
@@ -65,14 +67,14 @@ pub struct GetBlockTemplate {
     #[serde(rename = "coinbasetxn")]
     pub coinbase_txn: TransactionTemplate<amount::NegativeOrZero>,
 
-    /// Add documentation.
+    /// The expected difficulty for the new block displayed in expanded form.
     // TODO: use ExpandedDifficulty type.
     pub target: String,
 
-    /// Add documentation.
+    /// Last block time plus block spacing.
```

Comment:
The `min_time` consensus rule is:

```suggestion
    /// > For each block other than the genesis block, nTime MUST be strictly greater than
    /// > the median-time-past of that block.
    ///
    /// <https://zips.z.cash/protocol/protocol.pdf#blockheader>
```